evap/evaluation/models.py

```diff
@@ -110,11 +112,7 @@ def get_all_with_published_unarchived_results(cls):
 
     @classmethod
     def active_semester(cls):
-        return cls.objects.order_by("created_at").last()
-
-    @property
-    def is_active_semester(self):
-        return self == Semester.active_semester()
+        return cls.objects.filter(is_active=True).first()
```

Comment:
You could do 
```suggestion
        try:
            return cls.objects.get(is_active=True)
        except Semester.DoesNotExist:
            return None
```
to express that there should be at most one active semester but not multiple, but it's rather wordy, maybe it's not worth it.